### DISCUSSION TITLE
Error creating bean with name 'jobRepositoryTestUtils': Unsatisfied dependency expressed through method 'setDataSource' parameter 0

### DISCUSSION THREAD

**Discussion by oha85** [spring-projects/spring-batch#4380] [Q&A] [answered]
Hello,
I follow up [Issue-3767](https://github.com/spring-projects/spring-batch/issues/3767) to know if there is a workaround for spring-batch 2.7.2.
Many thanks in advance,
Best regards,

**Comment by fmbenhassine** [comment, marked as answer]
For v4, you have to specify a datasource if you want to use `@SpringBatchTest`. The workaround is to **not** use `@SpringBatchTest` and register the test utilities manually.
